skimage/filters/_gabor.py

```diff
@@ -43,6 +43,8 @@ def gabor_kernel(frequency, theta=0, bandwidth=1, sigma_x=None, sigma_y=None,
         deviations
     offset : float, optional
         Phase offset of harmonic function in radians.
+    no_DC_offset: bool, optional, default False
```

Comment:
I suggest to factor out the common code into a skimage internal function and then to create a separate `morlet_kernel` function. What do you think?